doc/guides/security-release-process.md

```diff
@@ -35,6 +35,8 @@ information described.
 
 ## Announcement (one week in advance of the planned release)
 
+* [ ] Verify that GitHub Actions are working as normal: https://www.githubstatus.com/.
```

Comment:
* [ ] Verify that GitHub Actions are working as normal: <https://www.githubstatus.com/>.